com.microsoft.mrtk.accessibility/Utilities/DescribableObject.cs

```diff
@@ -106,5 +117,34 @@ public DescribableFlag Flags
         /// "login button", "rocking chair", etc.
         /// </summary>
         public string Semantic => semantic;
+
+        #region Monobehaviour methods
+
+        private AccessibilitySubsystem accessibilitySubsystem = null;
+
+        private void Awake()
+        {
+            accessibilitySubsystem = XRSubsystemHelpers.GetFirstRunningSubsystem<AccessibilitySubsystem>();
```

Comment:
Could this happen in OnEnable? Maybe with a null check. Would help make it more robust to domain reloads (unless subsystems are serializable by default, but I haven't tried!)

Reply:
Or actually, we've tended to have a property/explicit backing field pattern for many subsystem lookups, where we access from the property which checks the backing field for null and tries to retrieve it if so.

Reply:
i can implement that pattern :)

Reply:
done 😄